packages/discord.js/src/managers/CategoryChannelChildManager.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const CachedManager = require('./CachedManager');
+const GuildChannel = require('../structures/GuildChannel');
+
+/**
+ * Manages API methods for CategoryChannels' children and stores their cache.
```

Comment:
```suggestion
 * Manages API methods for CategoryChannel's children and stores their cache.
```

Reply:
That’s not correct english

Reply:
Agree, its either `the CategoryChannel's` singular, or `CategoryChannels'` plural

Reply:
Got it, thanks explaining.